tests/phpunit/ServicesTests/UserGroupServiceTest.php

```diff
@@ -4,31 +4,38 @@
 
 namespace Tests\ServicesTests;
 
+use App\Contracts\UserAssignedGroupEmailContract;
 use App\Models\UserGroup;
 use App\Models\User;
 use App\Services\UserGroupService;
 use Illuminate\Foundation\Testing\DatabaseMigrations;
+use PHPUnit\Framework\Constraint\IsTrue;
 use Tests\TestCase;
 use Illuminate\Support\Facades\DB;
-
-
+use App\Contracts\UserGroupContract;
+Use Mockery;
 class UserGroupServiceTest extends TestCase
 {
     use DatabaseMigrations;
 
     protected $utility;
+    protected $userGroupUtility;
+    //protected $userAssignedGroupEmailUtility;
```

Comment:
remove the comment

Reply:
this line should be uncommented but still present